client/hotrod-client/src/main/java/org/infinispan/client/hotrod/iteration/KeyTrackerFactory.java

```diff
@@ -0,0 +1,21 @@
+package org.infinispan.client.hotrod.iteration;
```

Comment:
Since this class uses `impl` classes, shouldn't the package be: `org.infinispan.client.hotrod.impl.iteration` ?